Jenkinsfile

```diff
@@ -12,17 +12,22 @@ pipeline {
     stages {
         stage('clone') {
             steps {
-                git url: "https://github.com/ONSdigital/ras-integration-tests.git", branch: "testing-info-endpoints"
+                git url: "https://github.com/ONSdigital/ras-integration-tests.git", branch: "${env.BRANCH_NAME}"
             }
         }
         stage('pip install') {
             steps {
-                sh 'pip install -r requirements.txt'
+                sh 'pipenv install'
```

Comment:
Does this require the --dev flag?

Reply:
Since everything in this repo is related to testing should we have anything in dev-packages? We could just have it all in packages

Reply:
It could be dev packages or normal packages to be honest it probably doesn't matter much because it's not going to be deployed anywhere. Happy to put in dev packages if you think it makes sense